syntax = "proto3";

package text_process;

message NeuroRequest
{
    uint32 qid   = 1; // Некоторый произвольный идентификатор
    string query = 2;
}

message NeuroAnswer
{
    uint32 qid               = 1; // Соответствует NeuroRequest::qid
    repeated float embedding = 2;
    string error             = 3; // Заполняется, если произошла ошибка (embedding тогда пустой)
}
